t/cobrand/sutton.t

```diff
@@ -378,14 +379,26 @@ EOF
         is $report->state, 'fixed - council', 'A state change';
         my $update = FixMyStreet::DB->resultset("Comment")->order_by('-id')->first;
         is $update->photo, '34c2a90ba9eb225b87ca1bac05fddd0e08ac865f.jpeg';
-
+        $report->push_extra_fields( # Add extra fields expected on a Bulky waste report
+            { name => 'Bulky_Collection_Bulky_Items', value => '3::85::83'}, { name => 'Bulky_Collection_Notes', value => 'One::Two::Three' }
+        );
+        $report->update;
         FixMyStreet::Script::Alerts::send_updates();
-        my $body = $mech->get_text_body_from_email;
+        my $body = $mech->get_email->as_string;
```

Comment:
What was this change for? Surely the link should be present in the text part as well?

Reply:
It doesn't - finding it hard to track down the template for this, so left this as it is for the moment.

Reply:
The alert template? Is templates/email/default/alert-udate.txt / html, both of which contain the `problem_url` variable which is set in Alerts.pm to an /R/ URL if the alert_user_id is the user_id.

Reply:
The reason your regex wasn't finding the URL was because the text version of the email doesn't have a `"` at the end of it. Replacing that with e.g. `sutton.example.org(/R/[^"]*)` would match in the text version presumably.

Reply:
Yes, I see now, sutton has its own alert-update templates, and the text one does not include the link, but the HTML one still does by making the image of the update clickable.